Formula/confluent-platform.rb

```diff
@@ -15,10 +15,22 @@ def install
     prefix.install "bin"
     rm_rf "#{bin}/windows"
     prefix.install "etc"
+    prefix.install "libexec"
     prefix.install "share"
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/kafka-broker-api-versions --version")
+
+    home = ENV["HOME"]
+    system "echo", "#{home}/.confluent"
+    system "mkdir", "#{home}/.confluent/"
+    system "ls", "#{home}/.confluent"
+
+    ENV["CONFLUENT_HOME"] = "/usr/local/Cellar/confluent-platform/5.4.1/"
+    system "#{bin}/confluent", "--version"
+    assert_equal "0\n", shell_output("echo $?")
+    home = ENV["HOME"]
+    system "echo", "#{home}/.confluent"
```

Comment:
This isn't needed

Reply:
I was trying to get the build to work. Please read my comment below.

My change makes the install work. I have run and tested it on several machines. However, the build server fails the test due to a permission problem creating the "~/.confluent". This is due to the build environment. There was no test to start it using confluent.